code/controllers/subsystems/virtual_reality.dm

```diff
@@ -57,14 +57,20 @@
 		ckey = vr_mob.ckey
 		vr_mob.ckey = null
 		vr_mob.old_mob = null
-		vr_mob.languages = list(LANGUAGE_TCB)
+		if(istype(vr_mob, /mob/living/silicon))
+			var/mob/living/silicon/S = vr_mob
+			S.speech_synthesizer_langs = list(all_languages[LANGUAGE_TCB])
```

Comment:
if you're going to start doing this, it's better to make overrides on the proc itself for /mob/living/silicon/proc/body_return()